packages/helpers/src/product/getProductName.ts

```diff
@@ -1,17 +1,13 @@
 import { Product } from "@shopware-pwa/commons/interfaces/models/content/product/Product";
-import { getVariantOptionsLabel } from "./getVariantOptionsLabel";
 
 /**
- * @alpha
+ * @beta
  */
 export function getProductName({ product }: { product?: Product } = {}):
   | string
   | null {
   if (!product) {
     return null;
   }
-  const variantLabel = getVariantOptionsLabel({ product: product });
-  return `${product.translated.name}${
-    variantLabel ? " - " + variantLabel : ""
-  }`;
+  return product.translated?.name || product.name;
```

Comment:
Screams to me like we could introduce a monad which models the fallback `[callee].translated.[key] => [callee].[key]` for translated strings. => less redundant code.

Reply:
related issue was created https://github.com/vuestorefront/shopware-pwa/issues/1302